lib/sentry/plug.ex

```diff
@@ -82,16 +82,26 @@ defmodule Sentry.Plug do
     request_id_header = Keyword.get(env, :request_id_header, nil)
 
     quote do
-      defp handle_errors(conn, %{kind: kind, reason: reason, stack: stack}) do
-        opts = [body_scrubber: unquote(body_scrubber), header_scrubber: unquote(header_scrubber),
-                 request_id_header: unquote(request_id_header)]
-        request = Sentry.Plug.build_request_interface_data(conn, opts)
-        exception = Exception.normalize(kind, reason, stack)
-        Sentry.capture_exception(exception, [stacktrace: stack, request: request, event_source: :plug])
+      defp handle_errors(conn, error) do
+        opts = [body_scrubber: unquote(body_scrubber),
+                header_scrubber: unquote(header_scrubber),
+                request_id_header: unquote(request_id_header)]
+
+        {_, entry} = Sentry.Plug.capture(conn, error, opts)
```

Comment:
I exposed it as it would be nice to have possibility to write `handle_errors` by myself without digging too much into how `Sentry.Plug` works.